sequencer/src/catchup.rs

```diff
@@ -387,3 +387,44 @@ where
         &self.backoff
     }
 }
+
+/// Disable catchup entirely.
+#[derive(Clone, Copy, Debug, Default)]
+pub struct NullStateCatchup {
+    backoff: BackoffParams,
+}
+
+#[async_trait]
+impl StateCatchup for NullStateCatchup {
+    async fn try_fetch_accounts(
+        &self,
+        _instance: &NodeState,
+        _height: u64,
+        _view: ViewNumber,
+        _fee_merkle_tree_root: FeeMerkleCommitment,
+        _account: &[FeeAccount],
+    ) -> anyhow::Result<FeeMerkleTree> {
+        bail!("state catchup is disabled");
+    }
```

Comment:
We're using `NullStateCatchup` because we technically don't need to catchup on top of catchup correct?

Reply:
Yes, and if there's some logic error leading to recursive catchup I would rather it just error out than try something weird